.github/drafts/15938/15938_Audit_Logs_Outline.md

```diff
@@ -0,0 +1,88 @@
+# Audit logs
+
+Audit logs are a vital way to keep records of user activity related to app changes and platform settings. With access to this information, you can ensure that you're able to easily follow up and address any unexpected or undesirable events.
+
+{Screenshot of audit logs page}
+
+---
+
+## Logged events
+
+(Should we include the technical names of the events? Or maybe one example event per type? i.e. `datasource.created` etc.)
+
+There are several types of events tracked in Appsmith's audit logs:
+
+- App-specific activity
+    - Changes to pages/widgets (`page.updated`)
+    - Datasource configuration(`datasource.updated`)
+    - Running queries(`query.executed`)
+    - Deleting apps
+
+- Admin settings changes
+    - Who changed certain settings (`instance_setting.updated`)
+    - Changes/new groups or roles (`group.updated`)
```

Comment:
@spark-c Group related events will not be part of the immediate release

Reply:
So you can remove this line. It is not required.